### ISSUE TITLE
Code breaks when run with a new Stripe account

### ISSUE THREAD

**Issue by reshmabidikar** [killbill/killbill-stripe-plugin#123]
The stripe demo code breaks when it is run with a new stripe account.  

**Steps to reproduce:**

1. Create a new stripe account
2. Follow the steps documented [here](https://docs.killbill.io/latest/stripe_plugin)

The following error occurs:

```
com.stripe.exception.InvalidRequestException: The statement_descriptor parameter is not supported for the payment_method_type `card`. To continue supporting payment_method_type `card`, please pass in `statement_descriptor_suffix` or remove the `statement_descriptor` parameter altogether. Please see https://support.stripe.com/questions/use-of-the-statement-descriptor-parameter-on-paymentintents-for-card-charges for additional details
```

**Comment by pierre** [comment]
I wasn't able to reproduce it. I think this was fixed in 8.0.1 already (https://github.com/killbill/killbill-stripe-plugin/pull/118)?

**Comment by reshmabidikar** [comment]
Yes, I verified that this issue does not occur with version `8.0.1` of the stripe plugin. I'm closing this ticket.
